BunqSdk.Tests/Context/ApiContextTest.cs

```diff
@@ -12,13 +12,13 @@ public class ApiContextTest : BunqSdkTestBase, IClassFixture<ApiContextTest>
         /// <summary>
         /// Path to a temporary context file.
         /// </summary>
-        private const string CONTEXT_FILENAME_TEST = "context-save-restore-test.conf";
+        private const string ContextFilenameTest = "context-save-restore-test.conf";
 
-        private static ApiContext apiContext;
+        private static ApiContext _apiContext;
```

Comment:
Why the leading `_` ?

Reply:
I believe it was resharper doing some work 'automatically'. Shall I revert the underscore on all the private variables?

Reply:
Yes please revert the `_`, unless there is a style guide entry that is supporting this `_` 👍